server/src/enum.ts

```diff
@@ -469,7 +469,7 @@ export enum JobName {
   LIBRARY_SYNC_FILE = 'library-sync-file',
   LIBRARY_SYNC_ASSET = 'library-sync-asset',
   LIBRARY_DELETE = 'library-delete',
-  LIBRARY_QUEUE_SYNC_ALL = 'library-queue-sync-all',
+  LIBRARY_QUEUE_SCAN_ALL = 'library-queue-sync-all',
```

Comment:
You should also rename the actual enum value to avoid confusion.

Reply:
Thanks, I missed that one! Fixed.